### DISCUSSION TITLE
Alerts not showing up

### DISCUSSION THREAD

**Discussion by bird2473** [Security-Onion-Solutions/securityonion#7553] [Unsupported Versions]
**Setup:**
Standalone v2.3.110 (Home network)
ISO Install - VM on Proxmox
Manager/Node combined
so-status shows all services are OK
sudo salt-call state.highstate has no failures
Grid shows no failures (Status = OK)

I installed the previous version a couple weeks ago and was getting alerts from suricata right off the bat. I did a fresh install of the latest version v2.3.110 and have been having issues capturing traffic and generating alerts. I'm using a Mirrored port on my switch mirroring traffic on the uplink port.

**http://testmynids.org/uid/index.html** didn't generate any alerts

I did a **PCAP Import** from malware-traffic-analysis.net and can see the traffic in Kibana but no alerts from suricata.

**tcpdump** shows traffic on bond0 but it seems like mostly multicast traffic

**so-test**
Replaying PCAP(s) at 10 Mbps on interface bond0...
Actual: 111557 packets (12981286 bytes) sent in 10.38 seconds
Rated: 1249996.6 Bps, 9.99 Mbps, 10742.06 pps
Flows: 4102 flows, 394.99 fps, 2074477 flow packets, 45106 non-flow
Statistics for network device: bond0
        Successful packets:        55304
        Failed packets:            444
        Truncated packets:         0
        Retried packets (ENOBUFS): 0
        Retried packets (EAGAIN):  0

I'm going through the documentation again to see if I can find any other troubleshooting recommendations but I'm done for the night. Hoping someone can point me in the right direction. Thanks everyone.

**Comment by dougburks** [comment]
> **tcpdump** shows traffic on bond0 but it seems like mostly multicast traffic

Double-check your port mirroring configuration. Also try tcpdump on your management interface just to see if the mirror traffic might be going there instead.

**Reply by bird2473** [reply]
Thanks Doug

I think part of the problem was Proxmox. I changed the network config to passthrough the Network Interface and that started seeing the traffic.

I still didn't trigger any alerts when http://testmynids.org/uid/index.html - I tried accessing the link from a host on the network (i.e. my PC) and from the SecOnion CLI.

Anyways, progress to say the least. I'll spend more time on it this weekend.

Cheers

**Reply by dougburks** [reply]
Make sure that tcpdump is showing the full TCP stream when you access http://testmynids.org/uid/index.html.

Then make sure that SID 2100498 is enabled. 

Also note that if accessing http://testmynids.org/uid/index.html from a browser, repeated attempts may be served by cache. Usually better to use a command line tool like curl or wget.
